frontend/server/src/main/java/org/pytorch/serve/http/TokenAuthorizationHandler.java

```diff
@@ -85,7 +85,16 @@ public static void setupToken() {
             try {
                 token = new Token();
                 if (token.generateKeyFile("token")) {
-                    logger.info("Token Authorization Enabled");
+                    String loggingMessage =
+                            "######\n"
+                                    + "TorchServe now enforces token authorization by default.\n"
+                                    + "This requires the correct token to be provided when calling an API.\n"
+                                    + "Key file located at "
+                                    + ConfigManager.getInstance().getModelServerHome()
```

Comment:
Would suggest to add the filename as well to make sure users find the file right away.

Reply:
@udaij12 please include the key file name here so that it can be readily accessed by the user.